plugins/modules/meraki_mx_content_filtering.py

```diff
@@ -273,8 +279,12 @@ def main():
                 meraki.exit_json(**meraki.result)
             response = meraki.request(path, method='PUT', payload=json.dumps(payload))
             meraki.result['data'] = response
-            meraki.result['changed'] = True
-            meraki.generate_diff(current, response)
+            if recursive_diff(current, response) is None:
```

Comment:
What is the reason for converting from `is_update_required` to `recursive_diff`?

Reply:
Not quite from `is_update_required`. I have to change the `generate_diff` to `recursive_diff`, due to when the task did not change anything the `recursive_diff` was None. So the task had been ended with error (https://github.com/CiscoDevNet/ansible-meraki/issues/407#issuecomment-1374320425). So I think is faster way to change the `generate_diff` to `recursive_diff` due to if I have to do it in the `meraki` repo it will take more time etc :D